packages/data/README.md

```diff
@@ -19,8 +19,8 @@ _This package assumes that your code will run in an **ES2015+** environment. If
 Use the `registerStore` function to add your own store to the centralized data registry. This function accepts two arguments: a name to identify the module, and an object with values describing how your state is represented, modified, and accessed. At a minimum, you must provide a reducer function describing the shape of your state and how it changes in response to actions dispatched to the store.
 
 ```js
-const { apiFetch } = wp;
-const { registerStore } = wp.data;
+import apiFetch from '@wordpress/api-fetch';
+import { registerStore } from '@wordpress/data';
```

Comment:
I believe  we should remove "registerStore" and "registerGenericStore" from our docs and  just use "register" and "createReduxStore"

Reply:
Yes, it's a good idea. Let's do it separately as it requires more aggressive changes to README file.